doc/user/content/sql/create-source/webhook.md

```diff
@@ -157,3 +157,128 @@ After a successful secret creation, you can use the same secret to create differ
 ```
 
 Your new webhook is now up and ready to accept requests using the basic authentication.
+
+### Connecting with Segment
```

Comment:
Ah, yes! I was just opening an issue to write a Segment guide. To avoid stalling this PR, I'd be +1 to merging this PR and then take it within DevEx to restructure the content as multiple tool-specific guides (Segment, Rudderstack, Snowplow).